odoo/addons/base/views/res_users_views.xml

```diff
@@ -217,7 +236,34 @@
                                     <field string="Allowed Companies" name="company_ids" widget="many2many_tags" options="{'no_create': True, 'color_field': 'color'}"/>
                                     <field string="Default Company" name="company_id" context="{'user_preference': 0}"/>
                                 </group>
-                                <field name="groups_id"/>
+                                <group string="Groups">
+                                    <group>
+                                        <label for="group_ids" string="Selected groups"/>
+                                        <field name="group_ids" colspan="2" nolabel="1" context="{'search_default_filter_master_group': 1, 'search_default_filter_visible_category': 1}">
+                                            <list string="Selected groups" default_order="category_id,name" decoration-danger="id in parent.view_disjoint_group_ids" decoration-success="id in parent.view_implied_group_ids">
+                                                <field name="category_id" optional="hide"/>
```

Comment:
Always show, because names inside categories are generic (such as "user").
Another occurrence below.
```suggestion
                                                <field name="category_id"/>
```